### PR TITLE
first-responder: expand section on deploying dependency updates

### PR DESCRIPTION
**Pull request by ndushay**

### CODE REVIEW

File: infrateam_first_responder.md [hunk 1]
```diff
@@ -51,9 +51,16 @@ It *may* be that someone in a time zone further east than Palo Alto has already
...
 
+##### Merge 'em
 WIP script to automatically merge all dependency update PRs is currently in its own PR in access-update-scripts: https://github.com/sul-dlss/access-update-scripts/pull/104/files -- you can switch to the branch locally and use this script.  From the comments at the top, you will need a github access token.  Instructions are here:  https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line
 
-WIP Script to automatically deploy all sul-dlss PRs is here:  https://github.com/sul-dlss-labs/sdr-deploy
+##### Deploy 'em
+WIP Script to automatically deploy all sul-dlss PRs is here:  https://github.com/sul-dlss-labs/sdr-deploy.  Note that you will need to be sure you can ssh into each of the prod VMs from wherever you are running this script.
```
Comment (mjgiarlo) [inline infrateam_first_responder.md:58]: And not just the prod VMs but also QA and stage.

Should we also add a note here about projects which use novel environments? Thinking of hydra_etd UAT and sul_pub.

Comment (ndushay) [reply infrateam_first_responder.md:58]: sigh.  I'll revise.

Comment (ndushay) [reply infrateam_first_responder.md]: @mjgiarlo revised.

File: infrateam_first_responder.md [hunk 2]
```diff
@@ -51,9 +51,20 @@ It *may* be that someone in a time zone further east than Palo Alto has already
...
 
+##### Merge 'em
 WIP script to automatically merge all dependency update PRs is currently in its own PR in access-update-scripts: https://github.com/sul-dlss/access-update-scripts/pull/104/files -- you can switch to the branch locally and use this script.  From the comments at the top, you will need a github access token.  Instructions are here:  https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line
 
-WIP Script to automatically deploy all sul-dlss PRs is here:  https://github.com/sul-dlss-labs/sdr-deploy
+##### Deploy 'em
+WIP Script to automatically deploy all sul-dlss PRs using capistrano to local VMs is here:  https://github.com/sul-dlss-labs/sdr-deploy.  
+
+There are applications that need to be deployed separately, currently (6/9/2020) sinopia apps, dlme-transform, rialto-webapp, sul-pub.
```
Comment (mjgiarlo) [inline infrateam_first_responder.md:60]: How about rewording as:

```suggestion
There are applications that need to be deployed separately (i.e., not using `sdr-deploy`), currently (6/9/2020): sinopia apps, dlme-transform, and rialto-webapp are deployed using Terraform. Also note `sdr-deploy` is not a good tool for deploying the hydra_etd application to the `uat` environment nor for deploying the sul-pub application to its environments beyond `stage` and `prod`: https://github.com/sul-dlss/sul_pub/tree/master/config/deploy.
```

File: infrateam_first_responder.md [hunk 3]
```diff
@@ -51,9 +51,20 @@ It *may* be that someone in a time zone further east than Palo Alto has already
 
 It is a team task to complete these updates, but the first responder needs to make sure that all codebases needing updates have updates merged and deployed. Note that some projects may need to have PRs created by hand. It may be helpful to post in the `#dlss-infrastructure` Slack channel how many updates each developer should do, given who is working that day and how many PRs there are.
 
+##### Merge 'em
 WIP script to automatically merge all dependency update PRs is currently in its own PR in access-update-scripts: https://github.com/sul-dlss/access-update-scripts/pull/104/files -- you can switch to the branch locally and use this script.  From the comments at the top, you will need a github access token.  Instructions are here:  https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line
 
-WIP Script to automatically deploy all sul-dlss PRs is here:  https://github.com/sul-dlss-labs/sdr-deploy
+##### Deploy 'em
+WIP Script to automatically deploy all sul-dlss PRs using capistrano to local VMs is here:  https://github.com/sul-dlss-labs/sdr-deploy.
```
Comment (mjgiarlo) [inline infrateam_first_responder.md:58]: How about:

```suggestion
Use the `sdr-deploy` script to deploy all infrastructure projects (with exceptions noted below)  using capistrano to deployed environments is here:  https://github.com/sul-dlss-labs/sdr-deploy.  
```

**Comment by ndushay** [comment]
@mjgiarlo can you make another commit to the branch?  I'm embroiled in homebrew hell at the moment.

**Comment by mjgiarlo** [comment]
@ndushay done-zo. thanks!

File: infrateam_first_responder.md [hunk 4]
```diff
@@ -51,9 +51,20 @@ It *may* be that someone in a time zone further east than Palo Alto has already
 
 It is a team task to complete these updates, but the first responder needs to make sure that all codebases needing updates have updates merged and deployed. Note that some projects may need to have PRs created by hand. It may be helpful to post in the `#dlss-infrastructure` Slack channel how many updates each developer should do, given who is working that day and how many PRs there are.
 
+##### Merge 'em
 WIP script to automatically merge all dependency update PRs is currently in its own PR in access-update-scripts: https://github.com/sul-dlss/access-update-scripts/pull/104/files -- you can switch to the branch locally and use this script.  From the comments at the top, you will need a github access token.  Instructions are here:  https://help.github.com/en/github/authenticating-to-github/creating-a-personal-access-token-for-the-command-line
 
-WIP Script to automatically deploy all sul-dlss PRs is here:  https://github.com/sul-dlss-labs/sdr-deploy
+##### Deploy 'em
+Use the `sdr-deploy` script to deploy all infrastructure projects (with exceptions noted below)  using capistrano to deployed environments is here:  https://github.com/sul-dlss-labs/sdr-deploy.
```
Comment (jmartin-sul) [inline infrateam_first_responder.md:58]: would either go with:

"The `sdr-deploy` script to deploy all infrastructure projects (with exceptions noted below) using capistrano to deployed environments is here"

or

"Use the `sdr-deploy` script here to deploy all infrastructure projects (with exceptions noted below) using capistrano to deployed environments"

Comment (ndushay) [reply infrateam_first_responder.md:58]: also extra space between ")" and "using"

File: infrateam_first_responder.md [hunk 5]
```diff
@@ -51,9 +51,20 @@ It *may* be that someone in a time zone further east than Palo Alto has already
...
 
-WIP Script to automatically deploy all sul-dlss PRs is here:  https://github.com/sul-dlss-labs/sdr-deploy
+##### Deploy 'em
+Use the `sdr-deploy` script to deploy all infrastructure projects (with exceptions noted below)  using capistrano to deployed environments is here:  https://github.com/sul-dlss-labs/sdr-deploy.  
+
+There are applications that need to be deployed separately (i.e., not using `sdr-deploy`), currently (6/9/2020): sinopia apps, dlme-transform, and rialto-webapp are deployed using Terraform. Also note `sdr-deploy` is not a good tool for deploying the hydra_etd application to the `uat` environment nor for deploying the sul-pub application to its environments beyond `stage` and `prod`: https://github.com/sul-dlss/sul_pub/tree/master/config/deploy.
```
Comment (ndushay) [inline infrateam_first_responder.md:60]: colon instead of comma after ")" ?    Or better yet, make it a bulleted list?

File: infrateam_first_responder.md (hunk 4 above)
Comment (jermnelson) [reply infrateam_first_responder.md:58]: Should we include a sentence or two about needing to set SideKiq Pro credentials for Google Books and gis-robot-suite?

Comment (jmartin-sul) [reply infrateam_first_responder.md:58]: probably a good idea (though might also be good to do as a separate PR, just to get this merged)

File: infrateam_first_responder.md (hunk 5 above)
Comment (ndushay) [reply infrateam_first_responder.md:60]: don't hold up PR for these!

File: infrateam_first_responder.md (hunk 4 above)
Comment (ndushay) [reply infrateam_first_responder.md:58]: sidekiq pro keys should be a separate PR @jermnelson - let's get this one merged.
